src/stakingOptions.ts

```diff
@@ -364,35 +427,174 @@ export class StakingOptions {
   }
 
   /**
-   * Create an instruction for withdraw
+   * Create an instruction for exercise
    */
-  public async createWithdrawInstruction(
+  public async createExerciseReversibleInstruction(
```

Comment:
Naming is a bit confusing here. I might go with Exercise and UndoExercise

Reply:
dont want to override the existing to break compatibility.
withdraw is a little different since we are adding complexity under the hood

Reply:
these should match the program code, so if we want to change, should go back to the program code and change there

Reply:
It's fine just takes a bit to get familiar. Clarify in the comments above pls